realwork/realwork/DesignConstants/ViewDesigns.swift

```diff
@@ -118,3 +118,22 @@ class InversedPrimaryShadowView: ViewInversePrimaryGradient {
         self.layer.shadowOpacity = 0.9
     }
 }
+
+class Radius10View: UIView {
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        setup()
+    }
+
+    private func setup() {
+        self.layer.cornerRadius = 10
+        self.layer.maskedCorners = [.layerMaxXMinYCorner, .layerMinXMinYCorner,
+                                    .layerMinXMaxYCorner, .layerMaxXMaxYCorner]
+    }
+}
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Files should have a single trailing newline.](https://app.codacy.com/manual/NathanDVT/Musico/pullRequest?prid=5409503)